src/sql/properties.sql

```diff
@@ -3,27 +3,33 @@ select
     [Cores] = virtual_core_count, 
     [Memory] = CASE 
     WHEN hardware_generation = 'Gen4' THEN CONCAT(7 * virtual_core_count, ' GB')
+    WHEN hardware_generation = 'Gen5' THEN CONCAT(5.1 * virtual_core_count, ' GB')
+    WHEN hardware_generation = 'Gen6' THEN CONCAT(5.1 * virtual_core_count, ' GB')
+    WHEN hardware_generation = 'Gen7' THEN CONCAT(5.1 * virtual_core_count, ' GB')
+    WHEN hardware_generation = 'Gen8IM' THEN CONCAT(LEAST(7 * virtual_core_count, 560), ' GB')
+    WHEN hardware_generation = 'Gen8IH' THEN CONCAT(LEAST(13.6 * virtual_core_count, 870.4), ' GB')
     ELSE CONCAT(5.1 * virtual_core_count, ' GB')
     END,
     [Max storage] = CONCAT(max_storage_gb , ' GB'),
     [Service tier] = service_tier,
+    [Next-Gen General Purpose] = mf.[Next-Gen General Purpose],
     [Hardware generation] = hardware_generation, 
-	[Log write rate(max)] = 
+    [Log write rate(max)] =
     CASE 
-    WHEN service_tier = 'GeneralPurpose' AND virtual_core_count >= 8 THEN CONCAT(22, ' MB/s')
-    WHEN service_tier = 'GeneralPurpose' AND virtual_core_count <= 4 THEN CONCAT(virtual_core_count *3, ' MB/s')
-    WHEN service_tier = 'BusinessCritical' AND virtual_core_count <= 48/4 THEN CONCAT(virtual_core_count *4, ' MB/s')
-    ELSE '48 MB/s'
+    WHEN service_tier = 'GeneralPurpose' and mf.[Next-Gen General Purpose] = 'false' THEN CONCAT(LEAST(4.5 * virtual_core_count, 120), ' MB/s')
+    ELSE CONCAT(LEAST(4.5 * virtual_core_count, 192), ' MB/s')
```

Comment:
This limit was increased more than a year ago. For classic GP, on the instance level we have 4.5 x vCores with 120 CAP, and for Next-Gen and BC, same amount per vCore but 192 CAP